contributing/topics/maintainer-changelog.md

```diff
@@ -2,27 +2,27 @@
 
 > **Note:** When sending a Pull Request you should not include a changelog entry as a part of the Pull Request - this is to avoid conflicts. Contributors should not be concerned with updating the changelog as that is something only maintainers will do during merge.
 
-When a PR is merged it may or may not be included in the changelog. While most PRs deserve a changelog entry not every change should be included in the changelog as some have no user facing impact. Some examples of PRs that should not be included are:
+When a PR is merged it may or may not be included in the changelog. While most PRs deserve a changelog entry not every change should be included in the changelog as some have no user-facing impact. Some examples of PRs that should **not** be included are:
 
-- unit and acceptance test fixes
-- refactoring
-- documentation changes
+- Unit and acceptance test fixes
+- Refactoring
+- Documentation changes
 
 Otherwise, every PR that affects users should be added to the appropriate section:
 
 * `FEATURES` - new resources and data sources
-* `ENHANCEMENTS` - new properties, functionality, and features (including SDK/API upgrades)
+* `ENHANCEMENTS` - new properties, functionality, and features (including SDK/API upgrades), or anything that doesn't fit under the other two sections
```

Comment:
Would prefer to be specific about what goes under the `ENHANCEMENTS` section than adding an open ended categorisation. Can you elaborate on what you mean by this?
```suggestion
* `ENHANCEMENTS` - new properties, functionality, and features (including SDK/API upgrades), or anything that doesn't fit under the other two sections
```

Reply:
Yes, I considered adding it in case a contributor's change didn't fit neatly under one of three sections they would be able to default to one and not be blocked. But maybe that happens so infrequently it's not worth mentioning.